Projects/UOContent/Commands/Possess.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using Server.Commands.Generic;
+using Server.Mobiles;
+
+namespace Server.Commands
+{
+    internal static class Swap
+    {
+        private static readonly List<Layer> SPECIAL_LAYERS = new() { Layer.Backpack, Layer.Bank, Layer.Mount };
+
+        private static List<Item> GetItemsFromLayers(IEnumerable<Item> items) =>
+            new List<Item>(items).FindAll(item => !SPECIAL_LAYERS.Contains(item.Layer));
+        public static void Items(Mobile target, PlayerMobile caster)
+        {
+            var targetItems = GetItemsFromLayers(target.Items);
+            var casterItems = GetItemsFromLayers(caster.Items);
+
+            foreach (Item item in casterItems)
+            {
+                caster.RemoveItem(item);
+            }
+
+            foreach (Item item in targetItems)
+            {
+                caster.EquipItem(item);
+                target.RemoveItem(item);
```

Comment:
I don't think an item can be in two containers at once. Either you don't need to do a `RemoveItem` or it should be done before the equip.